<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recent_play</class>
 <widget class="QWidget" name="Recent_play">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>884</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>884</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_7" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#widget_7
{
	border-image: url(:/tupian/zhu.png);
}
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/tu/搜索.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="maximumSize">
             <size>
              <width>650</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
                border: 2px solid #ddd;
                border-radius: 15px;
	
	border-color: rgb(194, 194, 194);
                padding: 5px 10px 5px 30px;
                background: #f5f5f5;
	background-color: rgb(227, 227, 227);
                font-size: 14px;
            }
            QLineEdit:focus {
                border: 2px solid #0078d4;
            }</string>
            </property>
            <property name="placeholderText">
             <string>请输入要查询的歌手或歌曲名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton_2 {
	
	background-color: rgb(255, 24, 3);
    border: none; /* 去掉边框 */
    border-radius: 10px; /* 按钮圆角 */
    color: white; /* 字体颜色 */
    font-size: 16px; /* 字体大小 */
    padding: 10px 20px; /* 内边距 */
    text-align: center; /* 文字居中 */
    text-decoration: none; /* 去掉下划线 */
}

#pushButton_2:hover {
    background-color: #e55e00; /* 悬停时背景色 */
}
</string>
            </property>
            <property name="text">
             <string>播放全部</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="styleSheet">
        <string notr="true">
	border-image: url(:/tupian/zhu.png);
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableView" name="tableView">
            <property name="styleSheet">
             <string notr="true">
#tableView{
background-color: rgba(255, 255, 255, 0);
border:none
}
QHeaderView{ background: transparent; }</string>
            </property>
            <property name="autoScroll">
             <bool>true</bool>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
